fuse/context.py

```diff
@@ -231,106 +241,75 @@ def full_chain_context(
-def modify_s2_pattern_map(
-    s2_pattern_map, s2_mean_area_fraction_top, n_tpc_pmts, n_top_pmts, turned_off_pmts
+def xenonnt_fuse_full_chain_simulation(
+    output_folder="./fuse_data",
+    corrections_version=DEFAULT_XEDOCS_VERSION,
+    simulation_config=DEFAULT_SIMULATION_VERSION,
+    corrections_run_id=None,
+    clustering_method=None,  # defaults to dbscan, but can be set to lineage
+    fdc_map_mc=None,
+    cut_list=None,
+    **kwargs,
 ):
-    """Modify the S2 pattern map to match a given input AFT."""
-    if s2_mean_area_fraction_top > 0:
-        s2map = deepcopy(s2_pattern_map)
-        # First we need to set turned off pmts before scaling
-        s2map.data["map"][..., turned_off_pmts] = 0
-        s2map_topeff_ = s2map.data["map"][..., :n_top_pmts].sum(axis=2, keepdims=True)
-        s2map_toteff_ = s2map.data["map"].sum(axis=2, keepdims=True)
-        orig_aft_ = np.nanmean(s2map_topeff_ / s2map_toteff_)
-        # Getting scales for top/bottom separately to preserve total efficiency
-        scale_top_ = s2_mean_area_fraction_top / orig_aft_
-        scale_bot_ = (1 - s2_mean_area_fraction_top) / (1 - orig_aft_)
-        s2map.data["map"][..., :n_top_pmts] *= scale_top_
-        s2map.data["map"][..., n_top_pmts:n_tpc_pmts] *= scale_bot_
-        s2_pattern_map.__init__(s2map.data)
-    return s2_pattern_map
-
-
-# Probably not needed!
-@URLConfig.register("simple_load")
-def load(data):
-    """Some Documentation."""
-    return data
-
-
-@URLConfig.register("simulation_config")
-def from_config(config_name, key):
-    """Return a value from a json config file."""
-    config = straxen.get_resource(config_name, fmt="json")
-    return config[key]
-
-
-class DummyMap:
-    """Return constant results with length equal to that of the input and
-    second dimensions (constand correction) user-defined."""
-
-    def __init__(self, const, shape=()):
-        self.const = float(const)
-        self.shape = shape
-
-    def __call__(self, x, **kwargs):
-        shape = [len(x)] + list(self.shape)
-        return np.ones(shape) * self.const
-
-    def reduce_last_dim(self):
-        assert len(self.shape) >= 1, "Need at least 1 dim to reduce further"
-        const = self.const * self.shape[-1]
-        shape = list(self.shape)
-        shape[-1] = 1
-
-        return DummyMap(const, shape)
-
-
-@URLConfig.register("constant_dummy_map")
-def get_dummy(const, shape=()):
-    """Make an Dummy Map."""
-    itp_map = DummyMap(const, shape)
-    return itp_map
+    """Function to create a fuse full chain simulation context with the proper
+    settings for XENONnT simulations.
+
+    It takes the general full_chain_context and sets the proper
+    corrections and configuration files for XENONnT.
+    """
+
+    import fuse
+
+    # Get the simulation config file
+    if "/" in simulation_config:
+        # Local path to the simulation config file
+        simulation_config_file = simulation_config
```

Comment:
Maybe we are a bit more flexible if we check with something like: `os.path.isfile` if the config file is a local one or not.